doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -136,8 +136,8 @@ In the |NCS|, the provisioning data is generated as a hexadecimal file using the
 If Fast Pair is enabled with the :kconfig:option:`CONFIG_BT_FAST_PAIR` Kconfig option, the Fast Pair provision script is called automatically by the build system and the resulting hexadecimal file is automatically added to the firmware (that is, to the :file:`merged.hex` file).
```

Comment:
this is out of date and need updating for sysbuild

Reply:
This is not part of this PR

Reply:
@MarekPieta please send a PR with corrections

Reply:
The documentation content should be aligned across the whole NCS with one single PR

Reply:
I will create a PR with corrections